Packs/FeedAzureADConnectHealth/Integrations/AzureADConnectHealthFeed/AzureADConnectHealthFeed.py

```diff
@@ -0,0 +1,202 @@
+from CommonServerPython import *
+
+from typing import Any, Callable, Dict, List, Tuple
+
+import urllib3
+from bs4 import BeautifulSoup
+import re
+
+# disable insecure warnings
+urllib3.disable_warnings()
+
+INTEGRATION_NAME = 'Microsoft Azure AD Connect Health Feed'
+
+
+class Client(BaseClient):
+    """
+    Client to use in the Microsoft Azure Feed integration. Overrides BaseClient.
+    """
+
+    def __init__(self, base_url: str, verify: bool = False, proxy: bool = False):
+        """
+        Implements class for Microsoft Azure feeds.
+        :param url: the Azure endpoint URL
+        :verify: boolean, if *false* feed HTTPS server certificate is verified. Default: *false*
+        :param proxy: boolean, if *false* feed HTTPS server certificate will not use proxies. Default: *false*
+        """
+        super().__init__(base_url, verify=verify, proxy=proxy)
+
+    def build_iterator(self) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+        result = []
+        r = self._http_request('GET', url_suffix='', full_url=self._base_url, resp_type='text')
+
+        soup = BeautifulSoup(r, 'html.parser')
+
+        def subs(text):
+            """Cleans up the parsed entries.
+            Returns:
+                A string.
+            """
+            patterns = (('comp', 'com p'), ('comm', 'com m'), ('comf', 'com f'), ('\n', ''))
+            for e in patterns:
+                text = re.sub(e[0], e[1], text)
+            return text
+
+        try:
+            pattern = re.compile("(https?:\/\/|\*\.)(\w+\.|\w+-\w+\.){1,3}\w{2,3}")
+            scraped_urls: List = [subs(pattern.match(cell.text).group(0)) for cell in soup.select(  # type: ignore # noqa
+                "tbody tr td li") if pattern.match(cell.text)]
+            for url in scraped_urls:
+                result.append({
+                    'value': url,
+                    'type': FeedIndicatorType.DomainGlob if '*' in url else FeedIndicatorType.URL,
+                    'FeedURL': self._base_url
+                })
+
+        except requests.exceptions.SSLError as err:
+            demisto.debug(str(err))
+            raise Exception(f'Connection error in the API call to {INTEGRATION_NAME}.\n'
+                            f'Check your not secure parameter.\n\n{err}')
+        except requests.ConnectionError as err:
+            demisto.debug(str(err))
+            raise Exception(f'Connection error in the API call to {INTEGRATION_NAME}.\n'
+                            f'Check your Server URL parameter.\n\n{err}')
+        except requests.exceptions.HTTPError as err:
+            demisto.debug(str(err))
+            raise Exception(f'Connection error in the API call to {INTEGRATION_NAME}.\n')
+        except ValueError as err:
+            demisto.debug(str(err))
+            raise ValueError(f'Could not parse returned data to Json. \n\nError massage: {err}')
```

Comment:
the _http_request id performed beforehand, so this except block is not needed

Reply:
You are right. Do you want me to add a try to the _http_request?

Reply:
no need, it is being there, you can just rm this.

Reply:
Done